.github/workflows/auto-discovery.yml

```
@@ -0,0 +1,55 @@
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install pyyaml httpx
+
+      - name: Fetch VPS telemetry
+        env:
+          VPS_TELEMETRY_URL: ${{ secrets.VPS_TELEMETRY_URL }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:5:28: Quote this to prevent word splitting

(shellcheck)